### PR TITLE
Create Main component and Login component

### PR DESCRIPTION
**Pull request by crystallow1168**
**Main Component**
- Created Main component

**Login Component**
- Created email input, password input, and submit button
- Implemented email validation
- Implemented CSS
[Pending] Create password validation for a minimum length of 6 characters
[Pending] Create nav bar

### CODE REVIEW

File: client/src/pages/Login.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  },
+  loginText: {
+    fontSize: 35,
+    color: "black",
+    textAlign: "center",
+    padding: 30,
+  }
+}));
+
+const SubmitButton = withStyles(theme => ({
```
Comment (skassam21) [inline client/src/pages/Login.js]: This type of component should be in it's own file in a different folder for reusable components

File: client/src/pages/Login.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+const LoginContainer = withStyles(theme => ({
+  root: {
+    backgroundColor: "white",
+    width: 500,
+    height:400,
+    borderRadius: 10,
+    marginTop: 200,
+  }
+}))(Container);
```
Comment (skassam21) [inline client/src/pages/Login.js]: I feel like it's unnecessary to have this as a separate component and to just use a container with a classes.root inside the Login component 

Unless you are going to reuse this component for a Signup component, in which case you should generalize this component and make it reusable like i mentioend above

**Review by skassam21 — COMMENTED** [review]
Generally looks good, but I want some structuring around reusable components 

In terms of the styling, I'm fine with consistently using hooks for functional components and HOCs for class based components. The Main.css will be for styling that is wide across the whole app

File: client/src/pages/Login.js (hunk 2 above)
Comment (crystallow1168) [reply client/src/pages/Login.js]: I see! That makes much more sense!

File: client/src/pages/Login.js (hunk 1 above)
Comment (crystallow1168) [reply client/src/pages/Login.js]: Got it!

**Pull request merged**